prod_funding_sources.csv

```diff
@@ -0,0 +1,77 @@
+name,ror-identifier,award-number,file-id

```

Comment:
This is the data formatted for CSV.

Reply:
Just curious, where did the source for these come from? I see at least one line with no data, and a few with just file-id. Why were they included here?

Reply:
I took a excel file that Matt sent me and pulled out data that was needed for the metadata field. I'll attach that here so you can take a look. 

Honestly, we probably don't need to include the blank lines, but I figured it didn't hurt either and kept the function to access things consistent. No need to add a branch or anything.